src/main.rs

```diff
@@ -128,6 +148,108 @@ fn gen(cfg: &Config, markdown: String, out: impl Write) {
     }
 }
 
+fn ffmpeg<P: AsRef<Path>>(pdf: P, cfg: &Config) -> PathBuf {
+    // Start by demuxing the pdf into its frames, which gives us their number.
+    // We use poppler tools and not imagemagick because the latter is bloody stupid. It
+    // disables code access to pdf conversion as a 'security measure'. That's an idiots patch
+    // to broken code that just hurts usability. There's nothing malicious about what we're
+    // attempting to do but, I assumed, since they can't admit to the PHP crowds of SaaS image
+    // tools that their tools are utterly messy and completely unmaintainable they introduce a
+    // black-list that makes it impossible to use for other people as well.
+    // /rant
+    Command::new("pdftoppm")
+        .current_dir(&cfg.out_dir)
+        .arg("-png")
+        .arg(pdf.as_ref())
```

Comment:
```suggestion
        .args(&["-png", pdf.as_ref()])
```
I think grouping args here eases readability.

Reply:
This one is intentional. `pdf.as_ref()` is `&Path` while the former is `&str` so we would have to wrap either with more explicit converters which would, imo, negate some of the readability gained from grouping these arguments.